package.json

```diff
@@ -68,7 +68,7 @@
     "ng-ckeditor": "esvit/ng-ckeditor#^0.2.1",
     "ng-slide-down": "TheRusskiy/ng-slide-down#^1.0.0",
     "office-ui-fabric-react": "^4.35.2",
-    "ovh-angular-actions-menu": "^3.0.0",
```

Comment:
`yarn.lock` must be updated as well!

Reply:
Hello @antleblanc as I can see it seems correct in yarn.lock.
Can you check it pls ?

Reply:
Actually you are trying to bump `ovh-angular-actions-menu` from `v3.0.0` to `v3.2.0`. Must be reflected in diff on `yarn.lock` file isn't it?

Checking on `develop` branch and it's locked to `v3.1.1` (https://github.com/ovh-ux/ovh-manager-dedicated/blob/develop/yarn.lock#L4924).

Hope it's help you!

Reply:
@whispyy no problem! glad to help!